### PR TITLE
HWKMETRICS-516 Make batch size configurable in InsertedDataSubscriber

### PR DESCRIPTION
**Pull request by jshaughn**
- Now defaults to 100 but can be changed via configuration

### CODE REVIEW

File: alerting/alerting-jar/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import static org.hawkular.metrics.api.jaxrs.config.ConfigurationKey.DISABLE_METRICS_FORWARDING;
 import static org.hawkular.metrics.api.jaxrs.config.ConfigurationKey.DISABLE_PUBLISH_FILTERING;
 import static org.hawkular.metrics.api.jaxrs.config.ConfigurationKey.METRICS_PUBLISH_PERIOD;
+import static org.hawkular.metrics.api.jaxrs.config.ConfigurationKey.METRICS_PUBLISH_SIZE;
```
Comment (jshaughn) [inline alerting/alerting-jar/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java]: We aim to please.

**Comment by FilipB** [comment]
retest this please

**Comment by FilipB** [comment]
retest this please

**Comment by jshaughn** [comment]
@stefannegrea Please review.

**Comment by stefannegrea** [comment]
Thank you @jshaughn!

**Comment by FilipB** [comment]
This PR introduced perf drop 4%. It was missed because of env issue. It is now fixed by https://github.com/hawkular/hawkular-metrics/pull/672

**Pull request merged**